ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -84,9 +84,6 @@
 /* Accessibility label for when a task under progress has failed */
 "Accessibility.HUD.Failed" = "Failed";
 
-/* Accessibility label for when a task is in progress */
-"Accessibility.HUD.Loading" = "Loading";
```

Comment:
Yup! HUD now announced "In progress" from a different accessibility string

Reply:
i want to double check if we are reverting back the HUD if we need this again.

Reply:
how far back is it being reverted? the string usage was removed in #960 in the pure UIKit implementation of HUD (HUDView.swift)

Reply:
regardless, it should be removed either way as it's not being used currently. whenever the reverting happens, it can be readded/generalized to be shared

Reply:
yup we are planning to revert this entire swiftui implementation